k9mail/src/main/java/com/fsck/k9/view/MessageWebView.java

```diff
@@ -58,7 +58,6 @@ public void blockNetworkData(final boolean shouldBlockNetworkData) {
      */
     public void configure() {
         this.setVerticalScrollBarEnabled(true);
-        this.setVerticalScrollbarOverlay(true);
```

Comment:
Has this been tested on old versions of Android that we still support based on our minimum SDK?

Reply:
This has not been tested. I'm trying to now but I'm having trouble with the Android VM. About to remove this.